cdap-ui/app/cdap/components/Replicator/Detail/DetailContent/index.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useContext } from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import { Route, Switch, NavLink } from 'react-router-dom';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { basepath } from 'components/Replicator';
+import { DetailContext } from 'components/Replicator/Detail';
+import Overview from 'components/Replicator/Detail/Overview';
+import Monitoring from 'components/Replicator/Detail/Monitoring';
+import { humanReadableDate } from 'services/helpers';
+
+const styles = (theme): StyleRules => {
+  return {
+    root: {
+      display: 'grid',
+      gridTemplateColumns: '300px 1fr',
+      alignItems: 'end',
+    },
+    link: {
+      color: theme.palette.grey[50],
+      fontSize: '25px',
```

Comment:
Same here. We should use standardized headings. @leacd Is there any specific need to resize the headings here?

Reply:
the NavLink cannot apply the activeClass stylings if we use heading component here

Reply:
Wouldn't the Heading component be nested inside the NavLink? The activeClassStyles will still get applied to the H tags nested under the anchor tag right? Am I missing something.